src/main/java/at/porscheinformatik/seleniumcomponents/clarity/ClarityRadioContainerComponent.java

```diff
@@ -10,43 +13,45 @@
 /**
  * A clr-radio-container.
  *
- * @author ham
+ * @author ham, scar
  */
 public class ClarityRadioContainerComponent extends ClarityFormControlContainer
 {
+    private final SeleniumComponentListFactory<ClarityRadioComponent> radios =
+        new SeleniumComponentListFactory<>(this, selectByTagName("clr-radio-wrapper"), ClarityRadioComponent::new);
 
-    private final SeleniumComponentListFactory<ClarityRadioComponent> radioComponentFactory =
-        new SeleniumComponentListFactory<>(this, WebElementSelector.selectByClassName("clr-radio-wrapper"),
-            ClarityRadioComponent::new);
+    // ---
 
     public ClarityRadioContainerComponent(SeleniumComponent parent)
     {
-        super(parent);
+        super(parent, selectByTagName("clr-radio-container"));
     }
 
     public ClarityRadioContainerComponent(SeleniumComponent parent, WebElementSelector selector)
     {
         super(parent, selector);
     }
 
-    public SeleniumComponentListFactory<ClarityRadioComponent> getRadioComponentFactory()
```

Comment:
Please do not remove this method. If it is useless or should not be used mark it as deprecated.
Having access to the whole list might be useful in the future. You do the exact same thing in the Multiselect Combobox
https://github.com/porscheinformatik/selenium-components/pull/8/files#diff-ebf20d3423c14c07b39db039777584e1788310458b1f94f9a529530ece52314eR56

Reply:
Added it back. Not sure why I removed it... probably deemed it not needed. Sorry, that would be a breaking change. My mistake.